### ISSUE TITLE
Błąd podczas tworzenia paczki wysyłam z Allegro na środowisku testowym

### ISSUE THREAD

**Issue by lmichalFerrodo** [allegro/allegro-api#9602]
Występuje problem: 
 - [ ] z zasobem REST API / with a REST API resource
 - [ ] ze środowiskiem produkcyjnym / with the production environment
 - [X] ze środowiskiem testowym / with the test environment
 - [ ] z logowaniem lub autoryzacją / with login or authorization
 - [ ] na portalu https://developer.allegro.pl/ / at https://developer.allegro.pl/en/
 - [ ] inny / other

Dzień dobry,
podczas próby utworzenie przesyłki dla nowego zamówienia (a83eba20-53f9-11ef-9978-6ddd10b8d931) https://api.allegro.pl.allegrosandbox.pl/shipment-management/shipments/create-commands otrzymałem następujący komunikat błędu:

{"errors":[{"code":"COMMAND_EXISTS","message":"An order with the submitted ID already exists in the system","details":null,"path":"ae269a90-2cab-11ef-81d9-c7a80c0ae342","userMessage":"Zlecenie o podanym ID już istnieje w systemie","metadata":{}}]}

Uprzejmie proszę o pomoc.

**Comment by Lukasz-Zurek** [comment]
Z tego co widzę w logach, dla commandId: ae269a90-2cab-11ef-81d9-c7a80c0ae342 odnotowałem kolejną próbę utworzenia paczki. W przypadku [POST /shipment-management/shipments/create-commands](https://developer.allegro.pl/documentation/#operation/createNewShipment), dla każdej nowej przesyłki, którą tworzysz, musisz użyć nowego numeru UUID, lub jeśli go nie przekazujesz - wówczas wygenerujemy go za Ciebie i zwrócimy w odpowiedzi na żądanie, w polu **“commandId”**.

**Comment by lmichalFerrodo** [comment]
Dla pewności, jako commandId zawsze przekazywałem NUMER ZAMÓWIENIA. Czy z tego mógł wynikać problem?

**Comment by Lukasz-Zurek** [comment]
@lmichalFerrodo Tak, dokładnie. Pole **"commandId"** to identyfikator tego konkretnego żądania utworzenia paczki. Dla każdego żądania musi być unikalny i musi być w formacie [UUID](https://developer.allegro.pl/tutorials/informacje-podstawowe-b21569boAI1#identyfikatory-zasobow).

**Comment by Lukasz-Zurek** [comment]
@lmichalFerrodo W tej sytuacji w ogóle nie powinieneś przekazywać pola **“commandId”**, a w odpowiedzi zwrócimy Tobie ten identyfikator. Błąd zwracamy z powodu przekazania `""`.

**Comment by lmichalFerrodo** [comment]
Ok, udało się utworzyć paczkę do zamówienia 0c3919e0-548f-11ef-9978-6ddd10b8d931. Teraz otrzymuje błąd podczas sprawdzania statusu paczki z utworzonym przez Allegro commandId:

https://api.allegro.pl.allegrosandbox.pl/shipment-management/shipments/create-commands/?commandId=2bf3c5c8-b61e-4554-ad3d-4432327e9a4d

Błąd:
{"errors":[{"code":"SHIPMENT_NOT_FOUND","message":"Requested shipment was not found","details":"Requested shipment was not found","path":"shipmentId","userMessage":"Nie odnaleziono przewozu","metadata":{}}]}

**Comment by Lukasz-Zurek** [comment]
@lmichalFerrodo Źle odpytujesz: https://api.allegro.pl.allegrosandbox.pl/shipment-management/shipments/create-commands/?commandId=2bf3c5c8-b61e-4554-ad3d-4432327e9a4d

Powinieneś wysłać:

https://api.allegro.pl.allegrosandbox.pl/shipment-management/shipments/create-commands/2bf3c5c8-b61e-4554-ad3d-4432327e9a4d

**Comment by lmichalFerrodo** [comment]
Dziękuję, zmiana wywołania zadziałała.

Tutaj niestety musimy zrobić mały krok wstecz. 

Wywołując funkcje https://api.{environment}/shipment-management/delivery-services otrzymuje listę dostępnych serwisów. Wśród otrzymanych danych występują obiekty additionalServices oraz additionalProperties o następujących strukturach:
![obraz](https://github.com/user-attachments/assets/af9e4432-d2b6-4d00-beb8-6d42e49d8280)

Te same obiekty additionalServices oraz additionalProperties występują również w funkcji "Create new shipment" (https://api.{environment}/shipment-management/shipments/create-commands
) jednak tutaj nagle ich struktura jest inna.
![obraz](https://github.com/user-attachments/assets/cf33d1a6-3fce-40d9-a978-4a888a46f718)

Czy nie lepiej byłoby utrzymać jedną, wspólna strukturę tych obiektów?
Poniżej xml dla metody dostawy "Allegro One Box, One Kurier".

![obraz](https://github.com/user-attachments/assets/4845aeed-daeb-435f-8e4e-333d03926009)

Jak w tym przypadku powinien wyglądać obiekt additionalProperties przy tworzeniu nowej paczki? Ponieważ wartości o nazwach property1, property2 chyba nie działają.

![obraz](https://github.com/user-attachments/assets/4710e9be-a18c-4831-a244-0127d2a09ad1)

![obraz](https://github.com/user-attachments/assets/9bb092c7-c63e-4e00-bb35-daddfcdb2d07)

Uprzejmie proszę o odpowiedź.
